cmd/aspect/root/BUILD.bazel

```diff
@@ -13,7 +13,9 @@ go_library(
         "//cmd/aspect/clean",
         "//cmd/aspect/docs",
         "//cmd/aspect/info",
-        "//cmd/aspect/query",
+        "//cmd/aspect/query/aquery",
+        "//cmd/aspect/query/cquery",
+        "//cmd/aspect/query/query",
```

Comment:
I don't think they should live under `aspect/query/*`. They map to `aspect <cmd>`, so it makes more sense to have:

```
//cmd/aspect/aquery
//cmd/aspect/cquery
//cmd/aspect/query
```

Reply:
Moved. Would you want the commands under `pkg/aspect/query` to be moved as well? Those ones are grouped because they share a lot of code

Reply:
I think it makes sense to move those too. I haven't checked yet how they share code but we can always shuffle the shared code around, if necessary.

Reply:
Moved. `pkg/aspect/aquery` and `pkg/aspect/cquery` have been created. Shared code currently lives under `pkg/aspect/query/shared`